tests/integration/test_complete_run.cfg

```diff
@@ -64,17 +64,18 @@ years = "1850:1854:2",
 
 [tc_analysis]
 active = True
-scratch = "/lcrc/globalscratch/ac.forsyth2"
+scratch = "/lcrc/globalscratch/$USER"
 walltime = "00:30:00"
 years = "1850:1854:2",
 
 [e3sm_diags]
 active = True
 grid = '180x360_aave'
-obs_ts = '/lcrc/soft/climate/e3sm_diags_data/obs_for_e3sm_diags/time-series'
+obs_ts = '/lcrc/group/e3sm/diagnostics/observations/Atm/time-series'
 ref_final_yr = 2014
 ref_start_yr = 1985
-reference_data_path = '/lcrc/soft/climate/e3sm_diags_data/obs_for_e3sm_diags/climatology'
+# TODO: this directory is missing OMI-MLS
+reference_data_path = '/lcrc/group/e3sm/diagnostics/observations/Atm/climatology'
```

Comment:
@chengzhuzhang This directory is missing OMI-MLS. Is that ok?

Reply:
The data is here; https://web.lcrc.anl.gov/public/e3sm/diagnostics/observations/Atm/climatology/OMI-MLS/

Reply:
I see OMI-MLS in `/lcrc/group/e3sm/public_html/diagnostics/observations/Atm/climatology`. However, the `base_path` from Mache is `/lcrc/group/e3sm/public_html/diagnostic_output`, not `/lcrc/group/e3sm/public_html/diagnostics/`. So, OMI-MLS won't get picked up here. I think that's fine for a test though.